javascript/GridField.js

```diff
@@ -146,6 +152,37 @@
 			}
 		});
 
+		/**
+		 * Don't allow users to submit empty values in grid field auto complete inputs.
+		 */
+		$('.ss-gridfield .add-existing-autocompleter').entwine({
+			onbuttoncreate: function () {
+				var self = this;
+
+				this.toggleDisabled();
+
+				this.find('input[type="text"]').bind('keyup', function () {
+					self.toggleDisabled();
+				});
+			},
+			onunmatch: function () {
+				this.find('input[type="text"]').unbind('keyup');
+			},
+			toggleDisabled: function () {
+				var $button = this.find('button[type="submit"]'),
+					$input = this.find('input[type="text"]'),
+					inputValue = $input.val(),
+					buttonDisabled = $button.prop('disabled');
```

Comment:
yeah fair enough, i'll change it, it is a bit slower though. https://jsperf.com/prop-vs-ischecked/52